wrappers/leveldb/CMakeLists.txt

```diff
@@ -0,0 +1,55 @@
+cmake_minimum_required (VERSION 3.10)
+
+include_directories(${PROJECT_SOURCE_DIR}/include)
+
+include(ExternalProject)
+ExternalProject_Add(
+  leveldb
+  GIT_REPOSITORY https://github.com/google/leveldb.git
+  GIT_TAG master 
+  INSTALL_COMMAND ""  
+  BUILD_BYPRODUCTS ${CMAKE_CURRENT_BINARY_DIR}/leveldb-prefix/src/leveldb-build/${CMAKE_STATIC_LIBRARY_PREFIX}leveldb${CMAKE_STATIC_LIBRARY_SUFFIX}
+  CMAKE_ARGS -DCMAKE_CXX_FLAGS="-fPIC"
```

Comment:
Some flags should be added here. First, we want LevelDB to be built using the same compiler as PiBench.
If we compile PiBench by defining the compiler with: `$ CC=gcc-8 CXX=g++-8 cmake ..` we should pass these variables to the external project. This can be done by adding `-DCMAKE_CXX_COMPILER=${CMAKE_CXX_COMPILER} -DCMAKE_C_COMPILER=${CMAKE_C_COMPILER}` here.

Second, LevelDB allows us to skip the build of tests and benchmarks, which is useful in our case to speed the build process. So you can also add the following flags here: `-DLEVELDB_BUILD_TESTS=OFF -DLEVELDB_BUILD_BENCHMARKS=OFF`